src/main/java/org/spongepowered/api/util/command/CommandSource.java

```diff
@@ -44,6 +46,17 @@
     String getName();
 
     /**
+     * Gets the locale used by this command source. If this
+     * {@link CommandSource} does have a {@link Locale} configured or does not
+     * support configuring a {@link Locale}, the system default is used.
+     *
+     * @return The locale used by this command source
+     */
+    default Locale getLocale() {
+        return Locale.getDefault();
```

Comment:
Do remember that the server only has `en_US` (`Locale.US`) available, this should return `Locale.US` instead.